src/Domain/SizeOfElement.cpp

```diff
@@ -17,16 +17,15 @@
 #include "Utilities/StdArrayHelpers.hpp"  // IWYU pragma: keep
 
 template <size_t VolumeDim>
-void size_of_element(
-    const gsl::not_null<std::array<double, VolumeDim>*> result,
+std::array<double, VolumeDim> size_of_element(
```

Comment:
Because this function returns a lightweight object that doesn't require allocations, I don't see much benefit to using return-by-reference to avoid allocation. And return-by-value is just easier to use. I'll add a note to the commit message.